src/go/util/httppattern/sort_test.go

```diff
@@ -25,33 +25,6 @@ func parsePattern(pattern string) (string, string) {
 	return s[0], s[1]
 }
 
-func TestSortErrorHttpPattern(t *testing.T) {
```

Comment:
Why remove?

Reply:
Because right now the sort interface is changed from the url template strings to url template objects so the invalid ones are prevented when it is constructed, mainly in `service_info.go`